python/ray/_private/services.py

```diff
@@ -1650,7 +1652,8 @@ def start_raylet(redis_address,
     resource_argument = ",".join(
         ["{},{}".format(*kv) for kv in static_resources.items()])
 
-    gcs_ip_address, gcs_port = redis_address.split(":")
+    # TODO (iycheng): remove redis_ip_address after redis removal
+    redis_ip_address, redis_port = redis_address.split(":")
```

Comment:
Nice to include `ip` in the variable name.

Reply:
I'd prefer deleting them after redis removal.